build.gradle

```diff
@@ -132,10 +132,19 @@ publishing {
 			}
 		}
 	}
-	
+
 	repositories {
 		maven {
 			url "file:///itemapi_repo/"
 		}
 	}
 }
+
+/**
```

Comment:
Why did you use javadoc comments here? Just use normal multiline comments

Reply:
Gradle uses groovy, its okay i think?

Reply:
Okay i'll change.